torch/csrc/jit/mobile/flatbuffer_loader.cpp

```diff
@@ -196,7 +214,55 @@ std::unique_ptr<mobile::Function> FlatbufferLoader::parseFunction(
   return function;
 }
 
-at::Tensor FlatbufferLoader::parseTensor(
+IValue parseEnum(
+    FlatbufferLoader* loader,
+    const mobile::serialization::IValue* ivalue) {
+  const auto* enum_val = ivalue->val_as_EnumValue();
+  auto enum_type = loader->getOrCreateTypeAnnotations(enum_val->type_name())
```

Comment:
I'm not 100% sure so just asking: is EnumType even supported on edge?

Reply:
idk... pickle supports it though.